cyclonedx_py/client.py

```diff
@@ -186,6 +201,29 @@ def _get_input_parser(self) -> BaseParser:
             return EnvironmentParser()
 
         # All other Parsers will require some input - grab it now!
+        if not self._arguments.input_source:
+            # Nothing passed via STDIN, and no FILENAME supplied, let's assume a default by input type for ease
+            current_directory = os.getcwd()
+            try:
+                if self._arguments.input_from_conda_explicit:
+                    raise CycloneDxCmdNoInputFileSupplied('When using input from Conda Explicit, you need to pipe input'
+                                                          'via STDIN')
+                elif self._arguments.input_from_conda_json:
+                    raise CycloneDxCmdNoInputFileSupplied('When using input from Conda JSON, you need to pipe input'
+                                                          'via STDIN')
+                elif self._arguments.input_from_pip:
+                    self._arguments.input_source = open(os.path.join(current_directory, 'Pipfile.lock'), 'r')
+                elif self._arguments.input_from_poetry:
+                    self._arguments.input_source = open(os.path.join(current_directory, 'poetry.lock'), 'r')
+                elif self._arguments.input_from_requirements:
+                    self._arguments.input_source = open(os.path.join(current_directory, 'requirements.txt'), 'r')
+                else:
+                    raise CycloneDxCmdException('Parser type could not be determined.')
+            except FileNotFoundError as e:
+                raise CycloneDxCmdNoInputFileSupplied(
```

Comment:
@madpah as you are cating the `e` already, 
why not appending it to the error stash ala 
```py
raise CycloneDxCmdNoInputFileSupplied(''No input file was supplied and no input was provided on STDIN") from e
```